meson.build

```diff
@@ -1671,6 +1675,12 @@ if has_semihost
     set_variable(test_link_args_variable,
```

Comment:
I don't think any of the flags in this 'early' section are order dependent? Would it be possible to just change the order of parameters within this variable and then within the `_link_args` computation?

Reply:
Thanks, that works and simplifies the patch.